src/apps/components/AppsListPage/AppsListPage.tsx

```diff
@@ -33,34 +49,104 @@ const AppsListPage: React.FC<AppsListPageProps> = ({
   onAppInstallRetry,
   ...listProps
 }) => {
+  const styles = useStyles();
   const intl = useIntl();
+  // TODO Move to router
+  const [activeTab, setActiveTab] = useState<AppPageTabValue>(
+    AppPageTabValue.THIRD_PARTY,
+  );
 
   const appsInProgress = appsInProgressList?.appsInstallations;
 
+  const thirdPartyApps = useMemo(
+    () =>
+      installedAppsList?.filter(
+        app => !app.node.manifestUrl?.includes(".saleor.cloud"),
+      ),
+    [installedAppsList],
+  );
+
+  const saleorApps = useMemo(
+    () =>
+      installedAppsList?.filter(app =>
+        app.node.manifestUrl?.includes(".saleor.cloud"),
+      ),
+    [installedAppsList],
+  );
+
+  const renderContent = () => {
+    switch (activeTab) {
+      case AppPageTabValue.THIRD_PARTY: {
+        return (
+          <>
+            <p>
+              Third party apps are installed with App Manifests. They contain UI
+              accessible from dashboard and can extend it. Read more here.
```

Comment:
Strings should be defined using `defineMessages` and wrapped in `intl.formatMessage` or `<FormattedMessage />`